packages/integration-tests/projects/connect-popup/index.js

```diff
@@ -0,0 +1,370 @@
+                views: [
+                    { name: 'export-xpub', confirm: ['host'] },
+                ],
+            },
+            {
+                method: 'getAddress',
+                views: [
+                    { name: 'export-address', confirm: ['host'] },
+                    { name: 'check-address', confirm: ['device'] }
+                ],
+            },
+            // todo: forbidden key path at the moment
+            {
+                method: 'getAddress-multiple',
+                views: [
+                    { name: 'export-address', confirm: ['host'] },
+                ],
+            },
+            {
+                method: 'getAccountInfo',
+                views: [
+                    { name: 'export-account-info', confirm: ['host'] },
+                ],
+            },
+            // {
+            //     method: 'composeTransaction',
+            //     views: [
+            //         {
+            //             name: 'select-account',
+            //             confirm: [
+            //                 '#container > div > div.wrapper > div.select-account-list.segwit > button:nth-child(1)',
+            //             ]
+            //         },
+            //         {
+            //             name: 'select-fee-list',
+            //             confirm: ['.send-button', 'device', 'device']
+            //         }
+            //     ],
+            // },
+            // {
+            //     method: 'signMessage',
+            //     views: [
+            //         {
+            //             name: 'info-panel', // does not have a special screen
+            //             confirm: ['device', 'device']
+            //         },
+            //     ],
+            // },
+            // todo: wipe ends up on "connect device to continue" although device is connected
+            // note: probably only with bridge 2.0.31
+            // note: hmm not only :(
+            // {
+            //     method: 'wipeDevice',
+            //     views: [
+            //         {
+            //             name: 'device-management',
+            //             confirm: ['host', 'device']
+            //         },
+            //     ],
+            // },
+        ]
+    },
+    // {
+    //     device: {
+    //         wiped: true,
+    //     },
+    //     fixtures: [
+    //         {
+    //             method: 'resetDevice',
+    //             views: [
+    //                 {
+    //                     name: 'device-management',
+    //                     confirm: ['host', 'device']
+    //                 },
+    //             ],
+    //         },
+    //         {
+    //             method: 'recoverDevice',
+    //             views: [
+    //                 {
+    //                     name: 'device-management',
+    //                     confirm: ['host', 'device']
+    //                 },
+    //             ],
+    //         },
+
+
+    //     ]
+    // }
```

Comment:
Is this piece of commented code still required or is leftover?

Reply:
this is actually something newly added I would love to uncomment. the problem is that if there are more tests running, they start to be flaky and fail often. I havent found a source of that yet :(